tools/zstd/Makefile

```diff
@@ -11,25 +11,32 @@ PKG_LICENSE:=BSD-3-Clause
 PKG_LICENSE_FILES:=LICENSE
 PKG_CPE_ID:=cpe:/a:facebook:zstandard
 
-include $(INCLUDE_DIR)/host-build.mk
-include $(INCLUDE_DIR)/meson.mk
+HOST_BUILD_PARALLEL:=1
```

Comment:
Most likely unneeded, see https://github.com/openwrt/openwrt/pull/14971#issuecomment-2033829215

Reply:
In fact, it was already removed in the past:
https://github.com/openwrt/openwrt/commit/2e745e9be653626d37248c4f0a88ce9a5f90d183